<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QMainWindow" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebugWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Log</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QTextEdit" name="log">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <widget class="QComboBox" name="minLogLevel">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>0</y>
            <width>90</width>
            <height>26</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Trace</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Info</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warn</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Error</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>5</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Min logging level</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Drone</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="3">
        <widget class="QLabel" name="droneStateLabel">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QProgressBar" name="speed_y_pos">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Detection :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y speed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>298</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="detectionsLabel">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QGraphicsView" name="detections">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>420</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>X speed</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QProgressBar" name="speed_x_pos">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>State :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Position control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QPushButton" name="button_takeOff">
           <property name="text">
            <string>Take off</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_land">
           <property name="text">
            <string>Land</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="GamePad" name="positionControl" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Gamepad</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="GamePad" name="gamepad" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
<<<<<<< HEAD
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Log</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <widget class="QComboBox" name="minLogLevel">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>0</y>
            <width>90</width>
            <height>26</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Trace</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Info</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warn</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Error</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>5</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Min logging level</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>0</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Score:</string>
          </property>
         </widget>
         <widget class="QTextBrowser" name="score">
          <property name="geometry">
           <rect>
            <x>295</x>
            <y>1</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextEdit" name="log">
         <property name="readOnly">
          <bool>true</bool>
=======
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Position control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
>>>>>>> 0c730fcf3aa21d1c43046a5d1ec3808e65ce26cf
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Kad's PID</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Kp_z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Kd_z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="kd_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.005000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Ki_z</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="ki_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.005000000000000</double>
         </property>
         <property name="value">
          <double>0.025000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="kp_z">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.005000000000000</double>
         </property>
         <property name="value">
          <double>0.005000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Ki</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Kp</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Kd</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="kp">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>0.000500000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="ki">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>0.000500000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="kd">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>0.000500000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAwesome_GUI_Client">
    <property name="title">
     <string>Simon</string>
    </property>
   </widget>
   <addaction name="menuAwesome_GUI_Client"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamePad</class>
   <extends>QWidget</extends>
   <header>gamepad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
